saleor/graphql/checkout/tests/mutations/test_checkout_add_promo_code.py

```diff
@@ -543,6 +544,78 @@ def test_checkout_add_category_code_checkout_with_sale(
     assert checkout.discount_amount == Decimal(0)
 
 
+def test_checkout_add_variant_voucher_code_apply_once_per_order(
+    api_client, checkout_with_items, voucher_specific_product_type
+):
+    # TODO: update third variant price - include the third variant in discount
```

Comment:
How about it?

Reply:
Done, I will remove the comment. Thanks!